src/Rule/Typo.php

```diff
@@ -60,6 +60,8 @@ public static function getList(): array
             '/recalcuate/i' => null,
             '/achived/i' => null,
             '/overriden/i' => null,
+            '/succesfully/i' => null,
+            '/optionnally/i' => null,
```

Comment:
Should we also add "successfull" and "helpfull"?

Reply:
Yes, do you often face issues with misspelling here?

Reply:
yes, I have seen that several times

Reply:
@xabbuh Can you name some common misspellings I can match?

Reply:
besides "helpfull" and "usefull" I don't remember anything else

Reply:
ok